### PR TITLE
Add blog post for 1-26-2023

### PR DESCRIPTION
**Pull request by katebehrman**

### CODE REVIEW

**Comment by hongsupshin** [comment]
For reviewers, there are many files in change but most are just rendered html. You can just look at the `posts/index.qmd` for review.

File: posts/20230126/index.qmd [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* L3.3 Sampling bias in test distribution: This is choosing a non-representative subset for analysis.
+
+In general, categories L1.1-4 are clear cut and should be identifiable and avoidable. L2 is difficult because it depends on domain knowledge. Due to the nature of collaboration the researcher with this expert domain knowledge may not be the individual performing the ML analysis. Similarly, L3.1-3.3 are also difficult to detect and avoid if the researcher is using previously collected data and not designing an experiment and collecting data from scratch. In industry, data is often given to ML practitioners and there is no option to immediately collect more data or change the collection methodology. If this is the case, data leakage could be undetectable in many industry applications as ML practitioners are working with the best data at hand. In addition, there are limited strategies for identifying and mitigating these three types of data leakage.
```
Comment (athpud) [inline posts/20230126/index.qmd:34]: nice!

**Review by athpud — APPROVED** [review]

**Review by jafriyie1 — APPROVED** [review]
looks good to me :) !

File: posts/20230126/index.qmd [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+:::
+
+## Why this paper?
+Data leakage is a problem that we have all encountered as data scientists and scientific researchers. We just never knew how big! I was drawn to this paper because it attempts to quantify the impact of data leakage. It presents the unique view of considering the impact on ML-based science and shows how widespread and problematic data leakage is and leads to exaggerated claims of predictive performance. However, the frequency of data leakage and the impact of making these types of mistakes could be just as high or higher for ML-based industry applications. As a mixed group of ex ML-based science practitioners and current ML-based industry practitioners, I thought it would be insightful to discuss the eight types of data leakage identified, the proposed mitigation strategy of filling out model info sheets for ML-based science, and if the presented solution is also reasonable for a variety of ML-based industry applications.
```
Comment (hongsupshin) [inline posts/20230126/index.qmd:17]: Love this relatable intro!

File: posts/20230126/index.qmd [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+The first thing we noticed when looking at Table 1 was that several of the fields are areas with potential high-stakes such as medicine, clinical epidemiology, neuropsychiatry, medicine, and radiology. Engineering applications were excluded from this analysis but several fields sounded very close to this industry such as software engineering and computer security. We didn't feel that ML-based science should be differentiated from industry-based ML applications. If anything, we felt that data leakage in industry applications may be  harder to detect because they lack a formal peer review process and could have a bigger impact because these models are deployed in the real world.
+
+## Data leakage taxonomy
```
Comment (hongsupshin) [inline posts/20230126/index.qmd:24]: I like this short summary with our opinions included :)

File: posts/20230126/index.qmd (hunk 1 above)
Comment (hongsupshin) [inline posts/20230126/index.qmd:34]: "In industry, data is often given to ML practitioners and there is no option to immediately collect more data or change the collection methodology" -> Super important point

File: posts/20230126/index.qmd [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Model info sheets to mitigate data leakage
+We appreciated that the authors suggested a mitigation strategy. We found the questions in the model info sheets to be a useful sanity check for any ML practitioner. However, the level of detail requested from the info sheets would likely only be filled out if peer reviewed journals required it.  Even if journals require model info sheets as part of the peer review process, we agree with the authors that there are still limitations. Most notably, the claims in the model info sheets can't be verified without the code, computing environment, and data. There are some scientific research fields where authors might be unwilling or unable to disclose all information due to privacy concerns. In addition, there is nothing to prevent authors from lying or not having the ML expertise to fill out the model info sheet correctly. Since ML has become more accessible to non-ML experts, subtle cases might not easily be noticed by the author or reviewer.
```
Comment (hongsupshin) [inline posts/20230126/index.qmd:37]: Yeah, as we discussed, I wish they address more practical aspects of improvement as you mentioned here: "There are some scientific research fields where authors might be unwilling or unable to disclose all information due to privacy concerns."

**Review by hongsupshin — APPROVED** [review]
Great write-up! Thanks a lot, Kate. My comments are just strongly agreeing some parts in your writing. Not requesting any changes. 💯

**Pull request merged**